test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -870,6 +870,47 @@ public async Task UninstallPackagForIOSProjectInPMC(ProjectTemplate projectTempl
             }
         }
 
+        [NuGetWpfTheory]
+        [InlineData(ProjectTemplate.WCFServiceApplication)]
+        [InlineData(ProjectTemplate.NetStandardClassLib)]
+
+        public async Task InstallLatestPackageInPMC(ProjectTemplate projectTemplate)
+        {
+            EnsureVisualStudioHost();
+            using (var simpleTestPathContext = new SimpleTestPathContext())
+            {
+                // Arrange
+                var packageName = "InstallLatestInPMC";
+                var v100 = "1.0.0";
+                var v200 = "2.0.0";
+                await CommonUtility.CreatePackageInSourceAsync(simpleTestPathContext.PackageSource, packageName, v100);
+                await CommonUtility.CreatePackageInSourceAsync(simpleTestPathContext.PackageSource, packageName, v200);
+
+                using (var testContext = new ApexTestContext(VisualStudio, projectTemplate, XunitLogger, addNetStandardFeeds: true, simpleTestPathContext: simpleTestPathContext))
+                {
+                    var solutionService = VisualStudio.Get<SolutionService>();
+                    var nugetConsole = GetConsole(testContext.Project);
+
+                    // Act
+                    nugetConsole.Execute("install-package InstallLatestInPMC");
+                    testContext.SolutionService.Build();
```

Comment:
I assume this is needed because the WCF project is non-SDK style, and so build is used to trigger automatic restore.  Do our other tests have a more direct/explicit way of starting restore?  We don't validate anything related to build output, so the test could be a little faster if it doesn't do an unnecessary build.

Reply:
We need the building (line 896) and waiting for autorestore (line 897) since the assert for the NetStandardClassLibary project would fail to have the installed package info showing in the csproj file if we delete these two lines.